concent-vm/configure.yml

```diff
@@ -120,6 +120,7 @@
             src:        "{{ concent_deployment_dir }}/concent-vm/nginx-storage.service"
             dest:       "/etc/systemd/system/nginx-storage.service"
             remote_src: yes
+            mode:       0555
```

Comment:
Your commit description is too vague (Fix the problem with `nginx-storage.service`). It should say more about the problem. What exactly are you fixing?

Also, I think it should be a separate pull request because it's completely unrelated.

Reply:
Ok, I create separate pull request for this.